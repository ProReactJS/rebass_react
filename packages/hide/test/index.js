import React from 'react'
import TestRenderer from 'react-test-renderer'
import Hide from '../src'

const renderJSON = el => TestRenderer.create(el).toJSON()

test('Hide renders', () => {
  const json = renderJSON(
    <Hide
      xsmall
      small
      medium
      large
      xlarge
    />
  )

  expect(json).toMatchSnapshot()
  expect(json).toHaveStyleRule('display', 'none')
})
