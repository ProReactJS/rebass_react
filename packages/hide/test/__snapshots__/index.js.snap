// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@rebass/hide Hide renders 1`] = `
.c0 {
  display: none;
}

<div
  className="c0"
  display={
    Array [
      "none",
      "none",
      "none",
      "none",
      "none",
    ]
  }
/>
`;
